Extra/README.md

```diff
@@ -0,0 +1,235 @@
+Airport Challenge
```

Comment:
3 general improvements for next project

Try use more/ different dot notation 

Implement mocks for tests

stick to doing TDD approach (new for me)